src/classes/Model/Users/InvalidateToken.php

```diff
@@ -4,23 +4,23 @@
 
 use dhope0000\LXDClient\Model\Database\Database;
 
-class UpdateToken
+class InvalidateToken
 {
     public function __construct(Database $database)
     {
         $this->database = $database->dbObject;
     }
 
-    public function update(int $userId)
+    public function invalidate(int $userId)
```

Comment:
Why are we invalidating the tokens and not deleting them ? We will need to add a cron job to clear up this table once in a while if we don't delete (to be fair we may need an additional cron job as backup anyway)